site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -47,6 +47,65 @@ spec:
       seconds: 30
 ```
 
+{{% feature publishVersion="1.37.0" %}}
```

Comment:
In terms of ordering the page, the Counters and Lists examples should probably go after the `webhook` autoscaling example since they're still in Alpha.